packages/astro/src/content/config.ts

```diff
@@ -24,20 +24,20 @@ function getImporterFilename() {
 }
 
 // This needs to be in sync with ImageMetadata
-export type ImageFunction = () => ZodObject<{
-	src: ZodString;
-	width: ZodNumber;
-	height: ZodNumber;
-	format: ZodUnion<
+type ImageFunction = () => z.$ZodObject<{
```

Comment:
I think it should still be exported, pretty sure it was accessible to users in v5